docs/user-guide/consuming.rst

```diff
@@ -197,6 +197,40 @@ consumers to use if they need configuration options. Refer to the
 :ref:`conf-consumer-config` in the Configuration documentation for details.
 
 
+Monitoring
+==========
+
+A Fedora Messaging consumer can start a dedicated HTTP server to let users
+monitor its state. You can configure the port in the configuration file under
+the ``[monitoring]`` section, with ``address`` and ``port``.
+The default address is empty, which means that the monitoring server will
+listen on all interfaces. The default port is ``0``, which means that the
```

Comment:
This is a slightly surprising behavior since 0 usually means "bind to a free port" (try, for example, `nc -l ::1 0` and then see the port with `ss -plnt`).

Maybe just have the feature off if the section doesn't exist in the configuration and require both values to be set (or pick static defaults) if the section does exit.